backend/src/org/commcare/suite/model/graph/Graph.java

```diff
@@ -173,6 +185,27 @@ private void evaluateSeries(GraphData graphData, EvaluationContext context) {
                     }
                 }
                 graphData.addSeries(seriesData);
+
+                for (Enumeration<String> e = expandedConfiguration.keys(); e.hasMoreElements();) {
+                    String key = e.nextElement();
+                    String json = "";
+                    for (String pointValue : expandedConfiguration.get(key)) {
+                        json += "," + "'" + pointValue.replaceAll("'", "&apos;") + "'";
```

Comment:
You should use a `StringBuilder` since String are immutable and operating over them will always create new objects.